lib/pipeline.ts

```diff
@@ -165,6 +184,10 @@ export class Pipeline extends cdk.Construct {
 
     // emit an SNS notification every time build fails.
     this.addBuildFailureNotification(buildProject, `${props.title} build failed`);
+
+    if (props.autoBuild) {
```

Comment:
Not entirely clear to me what an "autobuild" is and why it needs a second build project.

Reply:
The first build project source is the pipeline and this one is github. It's a different build project...

Reply:
Aha, so maybe call it something like `PRBuild` ?

Reply:
Technically it’s not just for PRs. It also builds any commit to master and in the future we can add some configuration to control when it is triggered. It is automatic builds for all intents and purposes.